README.md

```diff
@@ -13,34 +13,154 @@ This plugin is available on [JetBrains Marketplace](https://plugins.jetbrains.co
 ![](docs/demo-workflow.gif)
```

Comment:
Not sure if new gif is good enough  :/
WDYT?

Reply:
I'd remove the moment where you FF `allow...` to match `build-chain`, this is complicates the situation since the yellow edge/fork point suggestion shows up etc.

Reply:
Also, the fact that `build-chain` has `PR #...` annotation despite being untracked... is kinda awkward ;) https://github.com/mkondratek/git-machete-blog-post/pull/1/files#r463688877

Reply:
Pls wait with regenerating this gif until right before merging this PR since some last-minute changes might happen